parl/remote/scripts.py

```diff
@@ -107,6 +107,9 @@ def cli():
 @click.option(
     "--cpu_num", type=int, help="Set number of cpu manually. If not set, it will use all "
     "cpus of this machine.")
+@click.option("--xpu", help="cpu for CPUs cluster, gpu for GPUs cluster.", default='cpu', type=str)
+@click.option(
+    "--gpu_ids", help="Set CUDA_VISIBLE_DEVICES manually. If not set, it will be empty string", default="", type=str)
```

Comment:
gpu_ids -> gpu.

help='comma separated list of GPU(s) to use. Default to use all available GPUs'